views/event/create.ejs

```diff
@@ -54,14 +81,34 @@
 
         <div class="input-group new-project-input">
             <div class="input-group-addon">&nbsp;Event Time: &nbsp;</div>
-            <input class="form-control" type="text" id="time" placeholder="Event Time">
+            <input class="form-control" type="date" id="time" placeholder="Event Time">
         </div>
 
         <div class="input-group new-project-input">
             <div class="input-group-addon">&nbsp;Event Location: &nbsp;</div>
             <input class="form-control" type="text" id="location" placeholder="Event Location">
         </div>
 
+        <div class="input-group new-project-input">
+            <div class="input-group-addon">&nbsp;Event Image Url: &nbsp;</div>
+            <input class="form-control" type="text" id="image" placeholder="Event Image Url">
+        </div>
+
+        <div class="input-group new-project-input">
+            <div class="input-group-addon">&nbsp;Event Description: &nbsp;</div>
+            <input class="form-control" type="text" id="description" placeholder="Event Description">
+        </div>
+
+        <div class="input-group new-project-input">
+            <div class="input-group-addon">&nbsp;Event Link: &nbsp;</div>
+            <input class="form-control" type="text" id="link" placeholder="Event Link">
+        </div>
+
+        <div class="input-group new-project-input">
+            <div class="input-group-addon">&nbsp;Event Type: &nbsp;</div>
+            <input class="form-control" type="text" id="type" placeholder="Event Type">
```

Comment:
Change this to an editable select, and give some default type like "Workshop, Kickoff, Group meeting" etc

Reply:
The way to have an editable select can be found in many places (we used an external script which is not difficult to add)